src/components/ha-expansion-panel.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+
+@customElement("ha-expansion-panel")
+class HaExpansionPanel extends LitElement {
+  @property({ type: String }) title = "";
+
+  @property({ type: Boolean }) expanded = false;
+
+  protected render(): TemplateResult {
+    return html`
+      <div
+        class="summary ${this.expanded ? "expanded" : ""}"
+        @click=${this._togglePanel}
+      >
+        <div class="summary-content">
+          <p class="summaty-title">
+            ${this.title}
+          </p>
+        </div>
+        <div class="summary-icon ${this.expanded ? "expanded" : ""}">
+          <svg viewBox="0 0 24 24">
+            <path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"></path>
+          </svg>
+        </div>
+      </div>
+      <div class="container ${this.expanded ? "expanded" : ""}">
+        <slot></slot>
+      </div>
+    `;
+  }
+
+  private _togglePanel(): void {
+    this.expanded = !this.expanded;
```

Comment:
We should fire an event here to notify that the panel expanded or collapsed

Reply:
I fire one of two: `expanded` or `collapsed`. Not sure of the naming convention in HA.